automation_tools/utils.py

```diff
@@ -96,6 +96,11 @@ def run_command(cmd=None):
         run(cmd)
 
 
+def version(satver='0'):
```

Comment:
This is functional for the intent, but points to the need for a more complex implementation to handle these versions.

Consider Robottelo's `SatVersionDependentValues` class, currently only used in host_info.py.  Its implementing some of the intended behavior of `Widgetastic.utils.VersionPick`, and needs to rely on comparing versions of Satellite.

I think we have a need for a version class that can parse the version string, and make comparisons of the components in the versions - like you've done here with the tuples.

Whether its here in automation-tools, or in robottelo itself, defining a Version class, instead of just a functions, allows for more direct comparison of the versions - without having to wrap them in a function call every time.

Check out this class in miq_version, which takes advantage of functools.total_ordering, and implements `__lt__` and `__eq__`. It parses the version string similarly to what you have done here.

Taking advantage of some of this and putting it together, instead of building these if/else trees to pick things relative to the satellite versions, we could define the dictionary of the options, and pick in a single statement, like:

```
ansible_repos = Widgetastic.utils.VersionPick(
    {
        Version.lowest(): 'rhel-{0}-server-ansible-2.6-rpms',
        '6.6': 'rhel-{0}-server-ansible-2.8-rpms'
    }
)
repos.append(ansible_repos.pick(current_sat_version_string))
```

Reply:
I dont want to pull in widgetastic. We keep `automation-tools` rather simple. This is just tool to install Satellite rather than full fledged automation suite.

Anyways this PR brings improvement over what's been there for years already.